server/src/metrics/actix_web_metrics.rs

```diff
@@ -187,12 +185,7 @@ impl RequestMetricsBuilder {
 
 /// construct meters for this crate
 fn get_versioned_meter(meter_provider: impl MeterProvider) -> Meter {
-    meter_provider.versioned_meter(
-        "unleash_edge",
-        Some(env!("CARGO_PKG_VERSION")),
-        Some(opentelemetry_semantic_conventions::SCHEMA_URL),
-        None,
-    )
+    meter_provider.meter("unleash_edge")
```

Comment:
Shouldn't we have open telemetry schema url and cargo pkg version here?

I found some docs indicating we can do:
```rust
    meter_provider.meter_with_scope(InstrumentationScope::builder("unleash_edge")
        .with_version(env!("CARGO_PKG_VERSION"))
        .with_schema_url(opentelemetry_semantic_conventions::SCHEMA_URL)
        .build())
```
but maybe this change is intentional

Reply:
No, you found what I was looking for. Updated to use your suggestion. Thanks :)